src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborReader.SkipValue.cs

```diff
@@ -2,28 +2,50 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
+using System.Diagnostics;
+
 namespace System.Security.Cryptography.Encoding.Tests.Cbor
 {
     internal partial class CborReader
     {
-        public void SkipValue()
+        // flag used to temporarily disable conformance level checks,
+        // e.g. during a skip operation over nonconforming encodings.
+        private bool _isConformanceLevelCheckEnabled = true;
```

Comment:
Since this is read outside of this partial, I'd recommend having it in the same file as the ctor.